### DISCUSSION TITLE
Faulting module name: libcef.dll, version: 103.0.8.0, App Crashes When Live But Not In Debug

### DISCUSSION THREAD

**Discussion by slane-ck** [cefsharp/CefSharp#4240] [General]
Hello,

My app works perfectly when debugging, but once we publish to live, it crashes as soon as the user does anything that accesses the browser. Any idea what might be causing this?

**Comment by amaitland** [comment]
Check the log file for errors, see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

Make sure all the require files were deployed with your application, see https://github.com/cefsharp/CefSharp/wiki/Output-files-description-table-%28Redistribution%29

Also make sure you've read https://github.com/cefsharp/CefSharp/discussions/4056

**Reply by slane-ck** [reply]
Thank you very much for your quick and helpful reply!

I checked the log file but I could only access it when running locally - is there a way for me to get such a file when running live? 

When running locally, I got the following error right at the top of the file: [0823/130844.648:ERROR:gpu_init.cc(486)] Passthrough is not supported, GL is disabled, ANGLE is . (I guess it got cut off) The same error also shows once further on, but that's the only error that is showing.

Most importantly, when checking the  required files I saw that several are missing: Required: icudtl.dat, snapshot_blob.bin,
v8_context_snapshot.bin. Optional: locales\*.pak, resources.pak, chrome_100_percent.pak, chrome_200_percent.pak,
vk_swiftshader_icd.json.

How do I safely get these files? I searched online but wasn't sure which site I could trust.

This isn't a ClickOnce app so I don't think https://github.com/cefsharp/CefSharp/discussions/4056 is relevant, unless it's also relevant to non-ClickOnce apps?

Thank you very much for your help!

**Reply by slane-ck** [reply]
Never mind, the files installed with Nuget, it turned out to be an issue with the way we're publishing. Thank you for your help!
